lib/src/query_builder.dart

```diff
@@ -0,0 +1,39 @@
+enum QueryType { singleIO }
+
+class QueryBuilder {
```

Comment:
what does it query? we will have two services so we may benefit if the name is more descriptive.